### ISSUE TITLE
MasterLBEnabled missing from clusters/results.go

### ISSUE THREAD

**Issue by iokiwi** [gophercloud/gophercloud#2117]
I recently got https://github.com/gophercloud/gophercloud/pull/2102 merged to add support for MasterLBEnabled fields.

However, it was pointed out that MasterLBEnabled is still missing from results.go
https://github.com/terraform-provider-openstack/terraform-provider-openstack/pull/1136#issuecomment-771787580

**Comment by jtopjian** [comment]
The PR is still marked as WIP. I will review it once it's ready. If you'd like to take over this work, I recommend forking the work done at #2118 so authorship of @iokiwi isn't lost and review our [contributor tutorial](https://github.com/gophercloud/gophercloud/tree/master/docs/contributor-tutorial) before submitting amendments.

**Comment by gadiego92** [comment]
But how was merged https://github.com/gophercloud/gophercloud/pull/2102? The `master-lb-enabled` field is missing on `results.go` file.

**Comment by jtopjian** [comment]
#2102 added support for `MasterLBEnabled` in requests. #2118 is adding support to the parsed JSON result. The two PRs go together, but nothing is broke by #2118 not being merged. I don't merge or review anything until requested.

**Comment by gadiego92** [comment]
Am I missing something? I have checked the files on the PR and it seems the changes are already done, right?

**Comment by jtopjian** [comment]
I might be misunderstanding. The patch in #2118 is still valid. That field has not been added to: https://github.com/gophercloud/gophercloud/blob/master/openstack/containerinfra/v1/clusters/results.go

**Comment by gadiego92** [comment]
I can see here (https://github.com/gophercloud/gophercloud/pull/2118/commits/d1feaa01be46d192963b002977b52ac6aa013564) the changes so just a merge is needed, right?

Sorry if I'm wrong I just want to help.

**Comment by jtopjian** [comment]
> I can see here d1feaa0 the changes so just a merge is needed, right? What's the next step?

In theory, yes. But the PR is marked as WIP (work in progress) and I have not gotten indication that this PR is ready to merge. I don't like to make assumptions and merge things without confirmation from the author.

If you are able to test the PR and confirm that it works, I can go ahead and merge it.

**Comment by gadiego92** [comment]
Ok. Now I got it. Thanks

I will try to test the PR and confirm it works but let's see because it's the first time I will contribute to a project.

**Comment by gadiego92** [comment]
@jtopjian I'm trying to run an acceptance test on the work done on https://github.com/gophercloud/gophercloud/pull/2118 but I get the following error:

```
$ go test -v -tags "fixtures acceptance" -run TestClustersCRUD
=== RUN   TestClustersCRUD
    convenience.go:35: Failure in clusters_test.go, line 17: unexpected error "You must provide exactly one of DomainID or DomainName to authenticate by Username"
--- FAIL: TestClustersCRUD (0.00s)
FAIL
exit status 1
FAIL    github.com/gophercloud/gophercloud/acceptance/openstack/containerinfra/v1       0.005s
```

I'm using OpenStack 5.2.0 and i just source the OpenStack RC file to work with it and there is not `DomainID ` nor `DomainName` on it.

Could it be that these was used in and older OpenStack version and it's not used anymore?

**Comment by jtopjian** [comment]
Try setting:

```
export OS_DOMAIN_ID=default
```

If that doesn't work, here's a working `openrc` example from a Devstack environment:

```
OS_REGION_NAME=RegionOne
OS_PROJECT_DOMAIN_ID=default
OS_DOMAIN_ID=default
OS_FLAVOR_ID=99
OS_CACERT=
OS_FLAVOR_ID_RESIZE=98
OS_POOL_NAME=public
OS_AUTH_URL=http://10.1.5.253/identity
OS_TENANT_NAME=demo
OS_USER_DOMAIN_ID=default
OS_USERNAME=admin
OS_VOLUME_API_VERSION=3
OS_EXTGW_ID=28d6a9db-6ff6-489f-8fb3-d64ab795dc4c
OS_PROJECT_ID=122ddc3141384ed4ab231162f07b2128
OS_NETWORK_ID=a1164afd-e6d0-4b49-97bf-e4d106fe8e1c
OS_AUTH_TYPE=password
OS_IMAGE_ID=bfe5e184-56b6-4952-82de-440197131c2a
OS_PROJECT_NAME=demo
OS_IMAGE_NAME=cirros-0.4.0-x86_64-disk
OS_PASSWORD=secret
OS_IDENTITY_API_VERSION=3
```
